components/raftstore-v2/src/operation/ready/snapshot.rs

```diff
@@ -313,4 +316,69 @@ impl<EK: KvEngine, ER: RaftEngine> Storage<EK, ER> {
         *snap_state = SnapState::Generated(snap);
         true
     }
+
+    pub fn on_applied_snapshot(&mut self) {
+        let mut entry = self.entry_storage_mut();
+        let term = entry.truncated_term();
+        let index = entry.truncated_index();
+        entry.set_applied_term(term);
+        entry.set_last_term(term);
```

Comment:
This is actually a false assumption that last term will be the same as truncated term.

Reply:
fixed